Framework/Core/src/ExternalFairMQDeviceProxy.cxx

```diff
@@ -254,6 +254,11 @@ void injectMissingData(fair::mq::Device& device, fair::mq::Parts& parts, std::ve
         continue;
```

Comment:
Ok, I think the problem is this one. If I matched something, I should still check for it, because if it has a wildcard it can still match something else.